src/GafferSceneUI/AttributeInspector.cpp

```diff
@@ -245,9 +248,47 @@ IECore::ConstObjectPtr AttributeInspector::value( const GafferScene::SceneAlgo::
 	return nullptr;
 }
 
-IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history ) const
+IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history, std::string &description ) const
 {
-	return history->scene->fullAttributes( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) )->member( m_attribute );
+	ScenePlug::PathScope pathScope( Context::current() );
+	ScenePlug::ScenePath currentPath( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) );
+	ConstObjectPtr inheritedAttribute;
+	if( currentPath.size() > 1 )
+	{
+		// We start the inheritance search from the parent in order to return the value that
+		// would be inherited if the inspected attribute did not exist at the original location.
+		currentPath.pop_back();
+
+		while( !currentPath.empty() )
+		{
+			pathScope.setPath( &currentPath );
+			auto a = history->scene->attributesPlug()->getValue();
+			if( const auto attribute = a->member( m_attribute ) )
+			{
+				inheritedAttribute = attribute;
+				description = "Inherited from " + ScenePlug::pathToString( currentPath );
+				break;
```

Comment:
We could just `return attribute` here, and remove `inheritedAttribute` and `if( inheritedAttribute )`.

Reply:
Changed and squashed to d1c5ac3.